src/jdk.internal.vm.ci/share/classes/jdk/vm/ci/hotspot/Cleaner.java

```diff
@@ -107,7 +107,7 @@ private static synchronized void remove(Cleaner cl) {
     /**
      * Remove the cleaners whose referents have become weakly reachable.
      */
-    static void clean() {
+    public static void clean() {
```

Comment:
Making this public avoids the need for a [method substitution](https://github.com/oracle/graal/blob/30492c3f7847a13ae7f8dc50663a5a039e49a8e7/substratevm/src/com.oracle.svm.graal.hotspot.libgraal/src/com/oracle/svm/graal/hotspot/libgraal/LibGraalSubstitutions.java#L112).